code/Extensions/SubmittedFileFieldExtension.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace SilverStripe\UserForms\Extensions;
+
+use SilverStripe\Assets\File;
+use SilverStripe\Core\Extension;
+use SilverStripe\Security\Security;
+
+class SubmittedFileFieldExtension extends Extension
```

Comment:
I don't think we should provide that extension out of the box. If we add it to our code base, then we're stuck supporting it. 

Maybe add it as an example to the doc and/or changelog for people who might prefer the old behaviour.